databricks/koalas/indexing.py

```diff
@@ -492,21 +494,35 @@ def __setitem__(self, key, value):
         rows_sel, cols_sel = key
 
         if (not isinstance(rows_sel, slice)) or (rows_sel != slice(None)):
-            raise SparkPandasNotImplementedError(
-                description="""Can only assign value to the whole dataframe, the row index
-                has to be `slice(None)` or `:`""",
-                pandas_function=".loc[..., ...] = ...",
-                spark_target_function="withColumn, select")
-
-        if not isinstance(cols_sel, str):
```

Comment:
Why removed?

Reply:
oh it moved to line number `514 ~ 518`

Reply:
ah, sorry i misunderstood.
Because now we should handle when cols_sel is list, but this line raises ValueError.
but now i think it maybe occurs side effect when `cols_sel` neither `str` or `list`
Then how about this?
```python
if not isinstance(cols_sel, (str, list)):
    raise ValueError("""only column names can be assigned""")
```

Reply:
oh i got it. and i figure out these whole logic that i've fixed have some problem from the start.
i'll fix them right away 😭